src/topology/algebra/ordered.lean

```diff
@@ -308,69 +310,62 @@ induced_orderable_topology' f @hf
 
 lemma nhds_top_orderable [topological_space α] [order_top α] [orderable_topology α] :
-  𝓝 (⊤:α) = (⨅l (h₂ : l < ⊤), principal {x | l < x}) :=
-by rw [@nhds_eq_orderable α _ _]; simp [(>)]
+  𝓝 (⊤:α) = (⨅l (h₂ : l < ⊤), principal (Ioi l)) :=
+by simp [nhds_eq_orderable (⊤:α)]
 
 lemma nhds_bot_orderable [topological_space α] [order_bot α] [orderable_topology α] :
-  𝓝 (⊥:α) = (⨅l (h₂ : ⊥ < l), principal {x | x < l}) :=
-by rw [@nhds_eq_orderable α _ _]; simp
+  𝓝 (⊥:α) = (⨅l (h₂ : ⊥ < l), principal (Iio l)) :=
+by simp [nhds_eq_orderable (⊥:α)]
 
 section linear_order
 
 variables [topological_space α] [linear_order α] [t : orderable_topology α]
 include t
 
-lemma mem_nhds_orderable_dest {a : α} {s : set α} (hs : s ∈ 𝓝 a) :
-  ((∃u, u>a) → ∃u, a < u ∧ ∀b, a ≤ b → b < u → b ∈ s) ∧
-  ((∃l, l<a) → ∃l, l < a ∧ ∀b, l < b → b ≤ a → b ∈ s) :=
-let ⟨t₁, ht₁, t₂, ht₂, hts⟩ :=
-  mem_inf_sets.mp $ by rw [@nhds_eq_orderable α _ _ _] at hs; exact hs in
-have ht₁ : ((∃l, l<a) → ∃l, l < a ∧ ∀b, l < b → b ∈ t₁) ∧ (∀b, a ≤ b → b ∈ t₁),
-  from infi_sets_induct ht₁
-    (by simp {contextual := tt})
-    (assume a' s₁ s₂ hs₁ ⟨hs₂, hs₃⟩,
-      begin
-        by_cases a' < a,
-        { simp [h] at hs₁,
-          letI := classical.DLO α,
-          exact ⟨assume hx, let ⟨u, hu₁, hu₂⟩ := hs₂ hx in
-            ⟨max u a', max_lt hu₁ h, assume b hb,
-              ⟨hs₁ $ lt_of_le_of_lt (le_max_right _ _) hb,
-                hu₂ _ $ lt_of_le_of_lt (le_max_left _ _) hb⟩⟩,
-            assume b hb, ⟨hs₁ $ lt_of_lt_of_le h hb, hs₃ _ hb⟩⟩ },
-        { simp [h] at hs₁, simp [hs₁],
-          exact ⟨by simpa using hs₂, hs₃⟩ }
-      end)
-    (assume s₁ s₂ h ih, and.intro
-      (assume hx, let ⟨u, hu₁, hu₂⟩ := ih.left hx in ⟨u, hu₁, assume b hb, h $ hu₂ _ hb⟩)
-      (assume b hb, h $ ih.right _ hb)),
-have ht₂ : ((∃u, u>a) → ∃u, a < u ∧ ∀b, b < u → b ∈ t₂) ∧ (∀b, b ≤ a → b ∈ t₂),
-  from infi_sets_induct ht₂
-    (by simp {contextual := tt})
-    (assume a' s₁ s₂ hs₁ ⟨hs₂, hs₃⟩,
-      begin
-        by_cases a' > a,
-        { simp [h] at hs₁,
-          letI := classical.DLO α,
-          exact ⟨assume hx, let ⟨u, hu₁, hu₂⟩ := hs₂ hx in
-            ⟨min u a', lt_min hu₁ h, assume b hb,
-              ⟨hs₁ $ lt_of_lt_of_le hb (min_le_right _ _),
-                hu₂ _ $ lt_of_lt_of_le hb (min_le_left _ _)⟩⟩,
-            assume b hb, ⟨hs₁ $ lt_of_le_of_lt hb h, hs₃ _ hb⟩⟩ },
-        { simp [h] at hs₁, simp [hs₁],
-          exact ⟨by simpa using hs₂, hs₃⟩ }
-      end)
-    (assume s₁ s₂ h ih, and.intro
-      (assume hx, let ⟨u, hu₁, hu₂⟩ := ih.left hx in ⟨u, hu₁, assume b hb, h $ hu₂ _ hb⟩)
-      (assume b hb, h $ ih.right _ hb)),
-and.intro
-  (assume hx, let ⟨u, hu, h⟩ := ht₂.left hx in ⟨u, hu, assume b hb hbu, hts ⟨ht₁.right b hb, h _ hbu⟩⟩)
-  (assume hx, let ⟨l, hl, h⟩ := ht₁.left hx in ⟨l, hl, assume b hbl hb, hts ⟨h _ hbl, ht₂.right b hb⟩⟩)
+lemma exists_Ico_subset_of_mem_nhds' {a : α} {s : set α} (hs : s ∈ 𝓝 a) {l : α} (hl : l < a) :
```

Comment:
I would rather use `Ioc` instead of `Ico` in the name of this lemma, as the `Ioc` is a subset, not the `Ioc`. This is even more visible in your lemma `exists_Ico_subset_of_mem_nhds`, in which there is no `Ico` at all...

Reply:
Done. I hope that this time I didn't mess up.